ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/Markdown.kt

```diff
@@ -67,9 +69,13 @@ internal fun Markdown(
     modifier: Modifier = Modifier,
     color: Color = Color.Unspecified,
     style: TextStyle = TextStyle.Default,
+    fontSize: TextUnit = TextUnit.Unspecified,
     fontWeight: FontWeight? = null,
+    fontFamily: FontFamily? = null,
+    horizontalAlignment: Alignment.Horizontal = Alignment.Start,
     textAlign: TextAlign? = null,
     allowLinks: Boolean = true,
+    textFillMaxWidth: Boolean = false, // This is to support V1 paywalls
```

Comment:
This is a bad hack... but I was running into issues... Basically `Markdown` was for some reason making each text use the `fillMaxWidth` modifier. This was probably wrong... When I removed it and I tried to fix it in all the templates I thought that in some cases, the view can potentially be modified with the change... So I wanted to avoid the UI change. The result is this flag to modify the behavior. We can remove this and revisit all the old templates and do what we think is right, but this was the "least risky" change. Lmk what you think.

Reply:
This seems fine to me 🤷‍♂️

Reply:
Maybe we can add a doc to the `Markdown` function explaining this parameter? Something like:

```kotlin
/**
 * @param textFillMaxWidth <explanation>
 */
internal fun Markdown(
```